bin/dev/create_helm_charts_pr.sh

```diff
@@ -53,6 +57,25 @@ rm -rf stable/uaa
 cp -r ../bundle/helm/cf stable/
 cp -r ../bundle/helm/uaa stable/
 
+# Show the places in the charts having RC references
+grep -rn '.-rc' stable/{cf,uaa}
+
+# Fixing RC references.
+
+# CLUSTER_BUILD: "2.20.2-rc1"
+sed -i 's|-rc.*"$|"|' stable/cf/values.yaml
+sed -i 's|-rc.*"$|"|' stable/uaa/values.yaml
```

Comment:
I didn't see this one, but that's probably dine.

Reply:
Note how that is not in the `Chart.yaml`, but the `values.yaml`. And not in the first page either. Somewhere deep under `env:`.

Reply:
Addressed in commit a38786f3eb1a4343fe7bb3a205775774729fa753.